specification/Overview.md

```diff
@@ -89,54 +142,89 @@ organised by data category (saveframe).
+    to obtain the same result.
 
-    11. Pseudoatom/wildcard names, stereospecific or non-stereospecific atom
+    12. Pseudoatom/wildcard names, stereospecific or non-stereospecific atom
     names can coexist for the same atom in the same file, but the proper name
     must be used with its defined meaning throughout the NEF file. It follows
     that when renaming the atoms to stereospecific entities, e.g. once the
     coordinates are known, the program must also maintain the name previously
     used to preserve continuity.
 
+      The chemical shift list must contain all known relevant shifts, but it is
+      not necessary to give the same chemical shift twice. If e.g. a Ser HB%
+      shift is given, the HBx, HBy, HB2, and HB3 are known to be the same.
+      Similarly HB% or QB can be used without any obligation to give their
+      shift explicitly, also if the atoms involved have different shifts. 
+      Indeed you can have restraints to atoms that have never
+      be measured (like oxygens in hydrogen bond restraints) and these need not
+      appear in restraint lists.
+      In cases where two protons are indistinguishable by NMR it
+      is strongly recommended to use the stereospecific atom names (HB2 and HB3)
+      in preference to the 'x' and 'y' forms. Since here may be cases where the
+      same atoms are stereospecifically assigned under some conditions but not
+      others (e.g. at different temperatures), it is not possible to guarantee
+      that only one of the two forms will be present.
+
       The RCSB has agreed to store the NEF atom identifiers in the tags
-      '_atom_site.auth_asym_id', '  _atom_site.auth_seq_id',
-      '_atom_site.auth_comp_id',  '_atom_site.auth_atom_id' . This preserves
+      '\_atom_site.auth_asym_id', '\_atom_site.auth_seq_id',
+      '\_atom_site.auth_comp_id',  '\_atom_site.auth_atom_id' . This preserves
       NEF naming, and gives a mapping for each structure model to the
-      equivalent RCSB tags '_atom_site.label_asym_id',
-      '_atom_site.label_seq_id', '_atom_site.pdbx_PDB_ins_code',
-      '_atom_site.label_comp_id', '_atom_site.label_atom_id'. The
+      equivalent RCSB tags '\_atom_site.label_asym_id',D
+      '\_atom_site.label_seq_id', '\_atom_site.pdbx_PDB_ins_code',
+      '\_atom_site.label_comp_id', '\_atom_site.label_atom_id'. The
       NEF sequence_code is a string, combined from a numerical sequence
       code and an  optional string insertion code, and therefore maps to two
       RCSB tags. Since the 'auth' tags can differ from one model to another
       within the same ensemble, this allows for floating stereospecific
       assignments that can vary between the individual models in the ensemble.
 
-    12. Residues and atoms that match the molecular sequence (including those
+    13. Residues and atoms that match the molecular sequence (including those
     using pseudoatom and non-stereospecific naming conventions) are understood
```

Comment:
Regarding 13.
This could be done in a better way by separating unassigned but observed chemical shifts form assigned chemical shifts. The proposed idea of ignoring what ever is inconsistent with the sequence is a nightmare for validation. As a worst case example you can take a sequence from one file and chemical shift from another file  and put together and claim this as a valid NEF file. It will pass the NEF file validation without any problem. No one is going to do that , but this is one of the worst case possibility.